features/application-title.yml

```
@@ -0,0 +1,9 @@
+name: application-title
+description: The `application-title` metadata name, used as `<meta name="application-title" content="">`, sets the text content that appears in the title bar of an installed web application.
```

The only other feature we have like this is structured a bit like this, with some edits for brevity. I don't know if it's _better_ but it is more consistent:

```suggestion
description: The `name="application-title"` attribute for the `<meta>` HTML element sets an installed web application's title bar text.
```